assets/scss/sections/pricing.scss

```diff
@@ -0,0 +1,73 @@
+@use '../utils' as *;
+@use '../vars' as *;
+
+.pricing {
+  text-align: center;
+  background-color: $lightgrey-bg;
+  @include flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 100px 0;
+  .pricing__intro {
+    padding: 0 20px 60px 20px;
+    @media (min-width: $extra-small) {
+      padding: 0 440px 60px 440px;
+    }
+    h1 {
+      font-size: 2rem;
+    }
+    p {
+      padding-top: 20px;
+    }
+  }
+}
+
+.pricing__cards {
+  color: white;
+  width: 90%;
+  @include flex;
+  flex-direction: column;
+  @media (min-width: $extra-small) {
+    flex-direction: row;
+    justify-content: center;
+  }
+  .pricing__card {
+    margin-top: 30px;
+    padding: 40px 0;
+    border-radius: 5px;
+    background: $pricing-card1;
+    background: linear-gradient(
+      180deg,
+      rgba(44, 215, 221, 1) 0%,
+      rgba(3, 104, 244, 1) 100%
+    );
+    @media (min-width: $extra-small) {
+      margin: 0 12px;
+      width: 25%;
+    }
+    .pricing__card-info {
+      text-align: left;
+      font-size: 0.7rem;
+      p {
+        padding: 15px 40px 0 40px;
+      }
+    }
+    p span {
```

Comment:
could use `.price` for this, not needed but makes it a bit more readable and less likely to brake when smth in html changes